website/source/docs/agent/options.html.md

```diff
@@ -883,6 +883,72 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
 * <a name="disable_keyring_file"></a><a href="#disable_keyring_file">`disable_keyring_file`</a> - Equivalent to the
   [`-disable-keyring-file` command-line flag](#_disable_keyring_file).
 
+* <a name="gossip_lan"></a><a href="#gossip_lan">`gossip_lan`</a> - (Advanced) This object contains a number of sub-keys
+  which can be set to tune the LAN gossip communications. These are only provided for users running especially large 
+  clusters that need fine tuning and are prepared to spend significant effort correctly tuning them for their
+  environment and workload.
+  
+  * <a name="gossip_nodes"></a><a href="#gossip_nodes">`gossip_nodes`</a> - The number of random nodes to send
+     gossip messages to per gossip_interval. Increasing this number causes the gossip messages to propagate 
+     across the cluster more quickly at the expense of increased bandwidth. The default is 3.
+  
+  * <a name="gossip_interval"></a><a href="#gossip_interval">`gossip_interval`</a> - The interval between sending
+    messages that need to be gossiped that haven't been able to piggyback on probing messages. If this is set to 
+    zero, non-piggyback gossip is disabled. By lowering this value (more frequent) gossip messages are propagated
+    across the cluster more quickly at the expense of increased bandwidth.
+  
+  * <a name="probe_interval"></a><a href="#probe_interval">`probe_interval`</a> - The interval between random node
+    probes. Setting this lower (more frequent) will cause the cluster to detect failed nodes more quickly 
+    at the expense of increased bandwidth usage.
+  
+  * <a name="probe_timeout"></a><a href="#probe_timeout">`probe_timeout`</a> - The timeout to wait for an ack from
+    a probed node before assuming it is unhealthy. This should be set to 99-percentile of RTT (round-trip time) on
+     your network.
```

Comment:
Should we specify the default? Also maybe the language is confusing "This should be set to <variable that might be unique to your DC>" implies that users _should_ fiddle with this when we don't really want to encourage that.

Maybe instead something like "This should be at least the 99-percentile of RTT in you network. The default is 500ms and is a conservative value suitable for almost all realistic deployments."